.github/workflows/docker.yml

```
@@ -0,0 +1,32 @@
+name: "docker"
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  bake:
+    name: "build and pushing convco docker image"
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+      - name: "Login to Docker Container Repository"
+        run: echo ${DOCKER_PASSWORD} | docker login -u ${DOCKER_USERNAME} --password-stdin
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      - name: "build docker image"
+        shell: bash
+        run: |
+          set -ex ;
+          export TAG=$(git describe --tags --abbrev=0 | sed 's/v//g') ;
```

in case this is master, the TAG should be latest

> The branch or tag ref that triggered the workflow run. For branches this is the format refs/heads/<branch_name>, and for tags it is refs/tags/<tag_name>.

```suggestion
          export TAG=$(echo ${{ github.ref }} | rev | cut -d/ -f1 | rev | tr -d 'v' | sed 's/master/latest/') ;
```

this will get the last part of the ref and drop the v or replace master with latest.